packages/backend/src/panels/ExploreGensPanel.ts

```diff
@@ -30,17 +39,8 @@ export class ExploreGensPanel extends AbstractWebviewPanel {
 
   public exploreGens: ExploreGens;
 
-  public constructor(context: vscode.ExtensionContext) {
-    super(context);
-    this.viewType = "exploreGens";
-    this.viewTitle = "Explore and Install Generators";
-    this.focusedKey = "exploreGens.Focused";
-    this.htmlFileName = path.join("exploregens", "index.html");
-    this.exploreGens = new ExploreGens(this.logger, this.isInBAS, this.context);
-  }
-
   public async loadWebviewPanel(uiOptions?: unknown) {
-    if (this.webViewPanel && _.isNil(uiOptions)) {
+    if (this.webViewPanel && isNil(uiOptions)) {
       this.webViewPanel.reveal();
     } else {
       await super.loadWebviewPanel(uiOptions);
```

Comment:
```suggestion
      super.loadWebviewPanel(uiOptions);
```

Reply:
no, this method is async and we need to wait till it finishes